### DISCUSSION TITLE
Show full name of the assignee rather than account username in GitHub Project Issues

### DISCUSSION THREAD

**Discussion by flavioaraujozup** [community/community#61959] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Oh Github Projects, I want to see the full name of the **assignee** who is responsible for developing that issue. In large organization it's very hard to find out who is by username.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by cyberfury10** [comment]
Hi, As a workaround, I have created an extension **WebEdit Pro -** ([Chrome extension](https://chrome.google.com/webstore/detail/webedit-pro/poacbppnkkaljgcfcgmabigfjhbkfdle), [Edge addon](https://microsoftedge.microsoft.com/addons/detail/webedit-pro/iglmlbhnbljiecdengccffmjlcgbjnei)) that can replace the user name with the desired full name.

Let's say usernames are random alphanumeric strings

K43234, K43235 & so on. Imagine there are 100 users under that repository with auto-generated alphanumeric IDs and whenever there is something happened in the repo. we will see

"K43234 committed a new change"
"K43235 raised a new PR"

With the above chrome extension & mapping as follows,

K43234 -> "John"
K43235 -> "Bruce"

The result would be:
"John committed a new change"
"Bruce raised a new PR"

I added capabilities to perform bulk edit (like in Postman), it is easy to add hundreds of username mapping at once

One can get the username and full name from the **_people_**  or **_Teams_** tab,

<img width="1657" alt="Screenshot 2023-08-24 at 2 10 11 PM" src="https://github.com/community/community/assets/45261873/73fcd2db-41e9-4a66-b646-128f37df744f">

here is the repo [link](https://github.com/cyberfury10/web-edit-pro) if you are interested to see the code & video [link ](https://www.youtube.com/watch?v=5j3OCeYzKFY)on how to use

**Comment by aryounce** [comment]
This is an issue for us as well. Many of our org members have usernames that are not easily associated to their real name.

**Reply by cyberfury10** [reply]
@aryounce 
I have been using the above extension for quite some time and it helps. here is video [link ](https://www.youtube.com/watch?v=5j3OCeYzKFY) on how to use

**Comment by myfoxtail** [comment]
This is quite critical for management who don't want to see assignees nicknames and prefer real names. Would be great to get this in as an optional setting in a project view.

**Comment by garmede** [comment]
If not using real name on GitHub profile, my suggestion is to prefix your ID with an @ in parallel.
I think the option to show real name is essential anyway.

**Comment by Deykun** [comment]
I had the same problem in my old company, instead of `@JimHelper` and `@MichaelScott` you would see `@dundermifflin_302512` and `@dundermifflin_302513`. I wrote a user script that learns the username on hover (from the profile's card) and swaps it to display the correct name in the future to solve that issue. The mentioned script: https://github.com/Deykun/github-usernames

Before:
![obraz](https://github.com/user-attachments/assets/5ccbf787-526c-4a1f-81e0-f78f367fd9da)

After:
![obraz](https://github.com/user-attachments/assets/8aa8f5ed-f0cb-4010-b0a3-d764f326429d)

**Reply by lunndal** [reply]
Anyone figured out any GitHub native way to solve this?
